### PR TITLE
Skeleton for Loan commands and stub for Serve mode in model

### PR DESCRIPTION
**Pull request by seeziyang**
Skeleton for Loan commands and stub for Serve mode in model

### CODE REVIEW

**Review by hoholyin — APPROVED** [review]
LGTM!

**Comment by Cronyxx** [comment]
LGTM!

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -94,4 +96,19 @@
 
     /** Returns the BorrowerRecords*/
     ReadOnlyBorrowerRecords getBorrowerRecords();
+
```
Comment (linyutinglyt) [inline src/main/java/seedu/address/model/Model.java:99]: Good and clear comments!

File: src/main/java/seedu/address/model/ModelManager.java [hunk 2]
```diff
@@ -181,11 +188,26 @@ public Model excludeBookBeingReplaced(Book toBeReplaced) {
...
+        // TODO
+        // return servingBorrower;
+        return Optional.of(new Borrower(new Name("Stub"), new Phone("12345"), new Email("mail@fakemail.co"))); // STUB
+    }
+
+    @Override
+    public boolean isServeMode() {
+        // TODO
+        // return servingBorrower.isPresent();
+        return true; // STUB
```
Comment (linyutinglyt) [inline src/main/java/seedu/address/model/ModelManager.java:208]: Good use of stub!

**Review by linyutinglyt — COMMENTED** [review]
LGTM!

HAND,
Yuting

**Pull request merged**